lib/src/data/datasource/remote/card_api_provider.dart

```diff
@@ -0,0 +1,31 @@
+import 'dart:io';
+import 'package:http/http.dart';
+import '../../../core/util/constants_api_calls.dart';
+
+class CardApiProvider {
+  final Client client;
+  int statusCodeOk = HttpStatus.ok;
```

Comment:
This is not necessary, you are creating a const that is a const itself

Reply:
Done.